### PR TITLE
added jetVersion to BuildInfo

### PR DESCRIPTION
**Pull request by gurbuzali**

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/instance/BuildInfo.java [hunk 1]
```diff
@@ -69,6 +76,18 @@ public byte getSerializationVersion() {
         return serializationVersion;
     }
 
+    public String getJetVersion() {
```
Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/instance/BuildInfo.java]: Can these values ever be null? What if someone isn't using JET?

Comment (gurbuzali) [inline hazelcast/src/main/java/com/hazelcast/instance/BuildInfo.java]: These values will be null if someone is not using JET

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/instance/BuildInfo.java]: Could you add this to the javadoc?

**Comment by pveentjer** [comment]
Apart from the javadoc and not yet having a green build; 👍

File: hazelcast/src/main/java/com/hazelcast/instance/BuildInfo.java [hunk 2]
```diff
@@ -35,14 +35,21 @@
     private final boolean enterprise;
     private final byte serializationVersion;
 
+    private final String jetVersion;
+    private final String jetBuild;
+    private final String jetRevision;
```
Comment (ahmetmircik) [inline hazelcast/src/main/java/com/hazelcast/instance/BuildInfo.java]: adding these fields to `toString` method can be helpful for diagnostic purposes.

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/instance/BuildInfo.java]: Good spotting @ahmetmircik

File: hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java [hunk 3]
```diff
@@ -180,6 +182,18 @@ public String convertToLetter(int size) {
         return parameterCreator.getParameters();
     }
 
+    private void addJetParameters(PhoneHomeParameterCreator parameterCreator, BuildInfo buildInfo) {
+        if (buildInfo.getJetVersion() == null) {
+            parameterCreator.addParam("jetv", "");
+            parameterCreator.addParam("jetb", "");
+            parameterCreator.addParam("jetr", "");
```
Comment (hasancelik) [inline hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java]: Could you update [PhoneHomeTest](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/test/java/com/hazelcast/util/PhoneHomeTest.java#L44) according to these changes? Default values are enough to test.

**Pull request merged**